static-analysis-plugin/plugin/src/test/groovy/com/novoda/staticanalysis/PmdViolationsEvaluatorTest.groovy

```diff
@@ -0,0 +1,44 @@
+package com.novoda.staticanalysis
+
+import org.junit.Before
+import org.junit.Test
+
+import static com.google.common.truth.Truth.assertThat
+import static com.novoda.staticanalysis.PmdViolationsEvaluator.PmdViolation
+
+public class PmdViolationsEvaluatorTest {
+
+    String xml = '''<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It was faster to copy/paste this, but I'm up for moving this in our fixtures/test resources.